### PR TITLE
got home thumbnails to work with Peder's template

### PR DESCRIPTION
**Pull request by limpa105**
# Pull Request Process
1. Ensure that your branch is as up-to-date with main as possible 
2. Fill out this template
3. Have at least 2 other people to get PR approval comments (could just add everybody on team)
4. Create PR

# Changes
## What changes did you make?
Got Thumbnails to show up on home page

# Verification
## How did you ensure that your changes work?
Tested Locally

# Screenshots

Screen Shot 2021-11-20 at 4 58 32 PM

### CODE REVIEW

File: source/home-page/assets/script.js [hunk 1]
```diff
@@ -61,47 +61,26 @@ let images = {
...
-      let ElemIngredient  = document.createElement("div");
-      ElemIngredient.id = "ingredients";
-      console.log(document.getElementsByClassName("container")[0]);
-      let place = 0; 
-      if (request == '30'){
-        place = 1;
+      let ele = document.createElement('search-recipe');
+      ele.data = vals.results[i];
+      let place = 0
+      if (request == 'Greek'){
```
Comment (nlpham) [inline source/home-page/assets/script.js:70]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445027)
```suggestion
      if (request == "Greek"){
```

File: source/home-page/assets/script.js [hunk 2]
```diff
@@ -61,47 +61,26 @@ let images = {
...
+        place = 1
       }
-      if (request == '60'){
+      if (request == 'Indian'){
```
Comment (nlpham) [inline source/home-page/assets/script.js:73]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445027)
```suggestion
      if (request == "Indian"){
```

File: source/home-page/assets/script.js [hunk 3]
```diff
@@ -61,47 +61,26 @@ let images = {
...
-      circle.id = "circle";
-      circle.textContent = "<" + recipe.readyInMinutes + "min";
-      element.appendChild(circle);
-      let ElemIngredient  = document.createElement("div");
-      ElemIngredient.id = "ingredients";
-      console.log(document.getElementsByClassName("container")[0]);
-      let place = 0; 
-      if (request == '30'){
-        place = 1;
+      let ele = document.createElement('search-recipe');
```
Comment (nlpham) [inline source/home-page/assets/script.js:67]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445027)
```suggestion
      let ele = document.createElement("search-recipe");
```

**Review by nlpham — APPROVED** [review]
Awesome! Perfect.

**Review by PederDDS — APPROVED** [review]
looking good!

**Pull request merged**